UI/src/view/dynamicAssetForm.js

```diff
@@ -907,6 +917,8 @@
                   rootElement.find('#feature-attributes-header').html(me.renderHeader(_assetTypeConfiguration.selectedLinearAsset));
                   rootElement.find('#feature-attributes-form').html(me.renderForm(_assetTypeConfiguration.selectedLinearAsset, isDisabled));
                   rootElement.find('#feature-attributes-footer').html(me.renderFooter(_assetTypeConfiguration.selectedLinearAsset));
+                    if(assetTypeConfiguration.lanePreview)
```

Comment:
"lanePreview" does not exist because nobody is using, yet. It works exactly as isSuggestedAsset attribute on point asset forms. For example in Droth 2262 road traffic law changes for traffic signs branch we used it but we had to do it ourselves now you only need to add to the asset type configuration of that asset type. Example on how to add this lane preview, in road work asset: 

```javascript
{
...
        layerName: 'roadWork',
        title: 'Tietyöt',
        newTitle: 'Uusi tietyöt',
        className: 'road-works-asset',
        isSeparable: true,
        allowComplementaryLinks: true,
        **lanePreview: true,**
...
}
```

Reply:
Ok ok, now it's explained.

Reply:
Talked with @marquesfm about this and the explanation was accepted so this continues as it is